### PR TITLE
Remove implicit frontend testing from `testing.run_app_in_cloud`

### PR DESCRIPTION
**Pull request by justusschock**
## What does this PR do?

This removes implicit frontend testing from run_app_in_cloud (that wasn't valid with recent frontend changes).

Unblocks #16650 

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [ x Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]
Thanks for this!

**Review by tchaton — CHANGES_REQUESTED** [review]
Just blocking until there is more context. Quite unclear why we remove the auth + admin view.

**Review by tchaton — APPROVED** [review]
LGTM !

File: tests/integrations_app/flagship/test_flashy.py [hunk 1]
```diff
@@ -73,6 +73,6 @@ def validate_app_functionalities(app_page: "Page") -> None:
 
 @pytest.mark.cloud
 def test_app_cloud() -> None:
-    with run_app_in_cloud(_PATH_INTEGRATIONS_DIR) as (admin_page, view_page, fetch_logs, _):
+    with run_app_in_cloud(_PATH_INTEGRATIONS_DIR) as (_, view_page, fetch_logs, _):
```
Comment (ethanwharris) [inline tests/integrations_app/flagship/test_flashy.py:76]:
```suggestion
    with run_app_in_cloud(_PATH_INTEGRATIONS_DIR) as (admin_page, view_page, fetch_logs, _):
```

**Pull request merged**